client/network/src/discovery.rs

```diff
@@ -196,10 +203,17 @@ impl DiscoveryConfig {
 			enable_mdns,
 			kademlia_disjoint_query_paths,
 			kademlia_protocols,
+			kademlia_replication_factor,
 		} = self;
 
 		let kademlia = if !kademlia_protocols.is_empty() {
 			let mut config = KademliaConfig::default();
+
+			if let Some(replication_factor) = kademlia_replication_factor {
+				dbg!(&kademlia_replication_factor);
```

Comment:
I'd say kill it or make it a regular log level

Reply:
Killing it.